README.md

```diff
@@ -139,18 +149,18 @@ Example from donate.mozilla.org:
 
 _Notes_: We want to keep the `trigger_welcome` at `N` and the `format` to `html`. We don't have the country info for now, but from what I understood, it's something we want to change.
 
-### Webhook Configuration
+## Review App
 
-There's a webhook endpoint for processing Braintree events. The events it supports are:
+Opening a PR will automatically create a Review App in the `donate-wagtail` pipeline. A slack bot posts credentials and links to Review Apps in to the `mofo-ra-donate-wagtail` channel.
 
-* `subscription_charged_successfully`
-* `subscription_charged_unsuccessfully`
-* `dispute_lost`
+This only work for Mo-Fo staff: you will need to manually open a Review App on Heroku for PRs opened by external contributors.
 
-The endpoint accepts requests on `/braintree/webhook/` and will verify the payload signature to ensure it's a legitimate event. [Documentation for Braintree webhooks can be found here](https://developers.braintreepayments.com/guides/webhooks/overview).
+## SSO and admin logins
```

Comment:
Do we need to note who to go to for new Auth0 accounts? Is it "File a Bugzilla ticket"?

Reply:
It is, but that bugzilla ticket would then come back to us going "this person is requesting access, is that okay?" so I'd recommend that for this we go with "ask us", and then if we agree those folks really need access, we file the bugzilla issue to request they get added to our donate-wagtail ldap group.

@cadecairos thoughts?

Reply:
Right, but how do _we_ remember that we need to file a Bugzilla ticket? 😄

Reply:
is there a designated "code owner" that we can say people should poke? (filing a bugzilla issue is effectively a given, since only moco has LDAP super powers)

Reply:
To be honest, In the interest of avoiding spear phishing attempts here, I'd rather not make the process "email somebody to ask for access"

Reply:
I was thinking "contact them on slack", but that's still a fair point.